libr/core/cmd.c

```diff
@@ -1260,6 +1274,193 @@ static int cmd_stdin(void *data, const char *input) {
 	return r_core_run_script (core, "-");
 }
 
+static void load_table(RCore *core, RTable *t, const char *file) {
```

Comment:
A small comment showing how a very simple table looks like? It's hard to review this function without seeing how a table is.

Reply:
really? arent tests enough for this?

Reply:
I don't see any test that shows how a table looks like. Could you point me to one?

Reply:
see https://github.com/radareorg/radare2/pull/17701/files#diff-057701ccb34a9d99f8fa2a0c0d53e59c

Reply:
also test/db/cmd/cmd_table